src/app/tests/TestAttributePersistenceProvider.cpp

```diff
@@ -198,20 +198,25 @@ void TestStorageAndRetrivalSignedScalarValues(nlTestSuite * inSuite, void * inCo
     persistenceProvider.Shutdown();
 }
 
-void TestGetNullFunctions(nlTestSuite * inSuite, void * inContext)
-{
-    NL_TEST_ASSERT(inSuite, AttributePersistenceProvider::GetNullValueForNullableType<bool>() == uint8_t(0xff));
-
-    NL_TEST_ASSERT(inSuite, AttributePersistenceProvider::GetNullValueForNullableType<uint8_t>() == uint8_t(0xff));
-    NL_TEST_ASSERT(inSuite, AttributePersistenceProvider::GetNullValueForNullableType<uint16_t>() == uint16_t(0xffff));
-    NL_TEST_ASSERT(inSuite, AttributePersistenceProvider::GetNullValueForNullableType<uint32_t>() == uint32_t(0xffffffff));
-    NL_TEST_ASSERT(inSuite, AttributePersistenceProvider::GetNullValueForNullableType<uint64_t>() == uint64_t(0xffffffffffffffff));
-
-    NL_TEST_ASSERT(inSuite, AttributePersistenceProvider::GetNullValueForNullableType<int8_t>() == int8_t(0x80));
-    NL_TEST_ASSERT(inSuite, AttributePersistenceProvider::GetNullValueForNullableType<int16_t>() == int16_t(0x8000));
-    NL_TEST_ASSERT(inSuite, AttributePersistenceProvider::GetNullValueForNullableType<int32_t>() == int32_t(0x80000000));
-    NL_TEST_ASSERT(inSuite, AttributePersistenceProvider::GetNullValueForNullableType<int64_t>() == int64_t(0x8000000000000000));
-}
+// This test is commented out because the GetNullValueForNullableType methods have been made private.
```

Comment:
Again, once you change to NumericAttributeTraits for the nullable stuff, you should be able to test whatever you want about it here, since it's public API....   Or just assume that the existing test coverage is good (which it probably is).

Reply:
I removed these tests once I changed to using NumericAttributeTraits.